src/Client/Pages/Authentication/ForgotPassword.razor.cs

```diff
@@ -0,0 +1,28 @@
+﻿using FSH.BlazorWebAssembly.Client.Infrastructure.ApiClient;
+using FSH.BlazorWebAssembly.Client.Shared;
+using Microsoft.AspNetCore.Components;
+
+namespace FSH.BlazorWebAssembly.Client.Pages.Authentication;
+
+public partial class ForgotPassword
+{
+    private readonly ForgotPasswordRequest _forgotPasswordRequest = new();
+    private CustomValidation? _customValidation;
+    public bool BusySubmitting { get; set; }
+
+    [Inject] public IIdentityClient _identityClient { get; set; } = default!;
+
+    public string _tenant { get; set; } = string.Empty;
```

Comment:
this can also be private, and _identityClient above as well... Not sure if _tenant even need to be a property?

almost there ;-)

Reply:
> this can also be private, and _identityClient above as well... Not sure if _tenant even need to be a property?

do you think a constant is needed in the appsettings.json file?

> almost there ;-)

Reply:
No, not in appsettings.json... I meant a constant in code... like we have now the "multitenancyconstants"... those should go to the shared lib...

Reply:
> No, not in appsettings.json... I meant a constant in code... like we have now the "multitenancyconstants" on the server... those should go to the shared lib...

so you mean an api call that downloads constants from the server?

Reply:
No, a copy of constants in the shared lib (that shared lib should actually become a link to the shared.dto's project on the server, but that still needs some work... for now we copy over the stuff to the local shared lib.

But anyway, this is less important... no need to take care of this now if you don't want to...